### DISCUSSION TITLE
How to access two levels with the same name in different hierarchies?

### DISCUSSION THREAD

**Discussion by chris-horril** [atoti/atoti#667] [Q&A] [answered]

**Comment by bloussou** [comment, marked as answer]
Hello,
When you have multiple levels with the same name in different hierarchies you need to provide a tuple `(<HIERARCHY_NAME>, <LEVEL_NAME>)` instead of just using the level name.
If I have two levels `Country`, one on hierarchy `Country` the other one on hierarchy `Geography` I have to do:
```diff
l = cube.levels
- l["Country"] # Will throw since two levels have the same name
+ l["Geography", "Country"] # From Geography hierarchy
+ l["Country", "Country"] # From Country hierarchy
```
